.github/workflows/tics.yml

```diff
@@ -31,9 +31,8 @@ jobs:
 
     - name: Install build dependencies
       run: |
-        sudo apt-get update
-        sudo apt-get install -y devscripts equivs lcov
-        mk-build-deps -s sudo -i
+        sudo --preserve-env apt-get install --yes devscripts equivs lcov ninja-build
```

Comment:
Could you please explain why `--preserve-env` was necessary?

Reply:
And why not update? Are these runner's apt cache guaranteed to be up to date?

Reply:
This is how Mir had it set up so I copied it, but I just tested and it isn't necessary. The variables that `--preserve-env` is protecting isn't related to building Multipass anyways.

Reply:
OK, makes sense. How about the missing `apt update`?

Reply:
We have this b/c:
https://github.com/canonical/mir/blob/df6674bd3686286aa0e422fdc26664e1fbef5adf/.github/workflows/tics.yml#L24-L25

You might get hit with some dependency that requires configuration, and the install will seem stuck b/c it will wait on user input.

Reply:
No reason to exclude it so I added it back.

Reply:
I see, interesting. Thanks @Saviq, is this only a concern in these runners? I suppose GH could be addressing that under the hood.

@sharder996, might be a good idea to do the same then (i.e. avoid apt-get asking what services should be restarted.)